docs/overview/low-level-data-format.md

```diff
@@ -1,7 +1,13 @@
 ---
 title: "Low-level Data Format"
 ---
-Substrate encodes data in the "Simple Concatenated Aggregate Little-Endian" (SCALE) data format, as implemented by the `parity-codec` crate and several JavaScript modules.
+Substrate encodes data in the "Simple Concatenated Aggregate Little-Endian" (SCALE) data format, as implemented by:
+* [parity-scale-codec](https://github.com/paritytech/parity-scale-codec) in Rust
+* [@polkadot/types](https://github.com/polkadot-js/api/tree/master/packages/types) in Javascript
```

Comment:
Dumb question, maybe @bkchr can answer, is there a spec for SCALE or should we say that the Substrate or Polkadot implementations are the reference?

Reply:
https://github.com/w3f/polkadot-re-spec/blob/master/polkadot_re_spec.pdf page 25ff

Reply:
That should probably live in a separate document if it's not Polkadot-specific. This should be docs for Substrate, and Polkadot is a Substrate-based chain.

Reply:
Might be worth opening a ticket fo that, but I don't think that this PR is the the right place to discuss where the SCALE specs should live. We should keep the comments helpful for @kaichaosun and don't go too far off-topic
From what I know the w3f Polkadot specs are the only real documentation that exists at the moment.

Reply:
Yeah that's what I figured. With a list of implementations, there should be some source of "truth," either a spec or reference implementation. That said, it's pretty obvious if you decode incorrectly and get a bunch of garbage.

Reply:
As discussed in the documentation channel. I would more likely see the specification in polkadot wiki and our wiki links to it. I already talked to Syed about this.